Tests/PublisherTests/DefaultPublisher/DefaultPublisherTests.swift

```diff
@@ -628,52 +628,91 @@ class DefaultPublisherTests: XCTestCase {
         
     }
     
-    func testPublisherWillRetryOnFailureOnSendEnhancedLocationUpdate() {
+    func testDefaultSkippedLocationsState() {
```

Comment:
This test is quite long and has multiple asserts which leads me to think that it could be divided into more small tests :wink:

Reply:
2e7c30a2d6e2aaad90c38ced5a0842537e0aa4dd